frontend/src/i18n/lang/pl-PL.json

```diff
@@ -289,7 +289,7 @@
 				"addedSuccess": "{type} został pomyślnie dodany.",
 				"updatedSuccess": "{type} został pomyślnie zaktualizowany."
 			},
-			"right": {
+			"permission": {
```

Comment:
Please don't change the polish translation in this PR, it will be overriden by Crowdin.